### PR TITLE
My feature

### PR DESCRIPTION
**Pull request by MasterLeiva**
- Created a new GitHub repository.
- Create a new branch according to GitHub's workflow.
- Add a .gitignore file to ignore node_modules and TEST.md
- I set up a linter for HTML and CSS.
- Added a descriptive README file to the project.
- Added 3 files (HTML, CSS, and TEST.md each one in a separate commit):
- HTML includes a header (h1) with the text "Hello Microverse!". 
- CSS style with my favorite color
- TEST.md file includes the text "This file should be ignored by git".

### CODE REVIEW

**Comment by MasterLeiva** [comment]
For some reason my tests are failing, I don't know why though.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+test.md
```
Comment (DuaneDave) [inline .gitignore]: Good job including the test.md file in here 👍. However, the file is still appearing the repository. I suggest that you resolve this concern in order to correctly implement the usage of a .gitignore file.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @MasterLeiva 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Correct usage of GitHub flow
✔️ Linters are passing

## Required Changes ♻️
- Please ensure that the linter checks are passing as they are there to help you catch syntax errors that you may not have notice. Confirm that the webhint and stylelint checks are green before requesting for another.

linter


- It is very important to include a very descriptive summary of the activities you have done in this milestone in the PR summary as they aid a code reviewer to know what you have accomplished. Use the icon I have circled out in the screenshot to make edits to the PR summary.
OR


- When I click on the MIT link at the bottom of your readme, it is expected to redirect me to your License file but currently, it leads to a 404 page. I suggest resolving this issue to make the readme more professional.

This is the section I speak about :
mit

To resolve the above issue, edit the readme by changing [MIT](./LICESE) to [MIT](./MIT.md)
editMIT


- You have removed an important section of the readme which is the deployment section. Here you can add platform where this project can be deployed or commands that can be use. If there is none you can put N/A in that section. Just make sure it exists and its not empty 😉

deployment
- Lastly, the live demo link is not available and as such I recommend putting a text that says coming soon or not available since you do not have a live link.


_Check the comments under the review._

## Optional suggestions
- I am making this optional because you can not go back to changing commit messages. However, you should know that commit messages should be written in an imperative form with the first letter capitalized in order to follow best practices.

Instead of saying `Fixed lighthouse node version` you can say `Fix lighthouse node version`

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @MasterLeiva  ,

Great job so far! 

May I highlight the following: 
- [x] Your Readme looks fine👏  ✔️ 
- [x] The project is set correctly✔️ 
- [x] No Linter errors✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Comment by MasterLeiva** [comment]
Thank you! 🙌🏼

**Pull request merged**